front/src/pages/room/index.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  ChatBox,
+  ImgBox,
+  MessageBox,
+  MiddleBox,
+  MsgBox,
+  MsgWrapper,
+  NameBox,
+  Time,
+  TimeDiv,
+  Wrapper,
+  NameField,
+  FormBox,
+  MiddleDiv,
+  BackBtn,
+  ButtonStyle,
+  Input,
+} from './style';
+import { useRef } from 'react';
+import { useParams, useNavigate } from 'react-router';
+import io, { Socket } from 'socket.io-client';
+import { useEffect, useState } from 'react';
+import gravatar from 'gravatar';
+import { IUserInfo } from 'interface';
+const socket = io(`http://localhost:3065`);
+const Room = () => {
+  const { id } = useParams();
+  let navigator = useNavigate();
+  const scrollRef = useRef<HTMLInputElement>(null);
+  const [input, setInput] = useState('');
+  const [messageObj, setMessageObj] = useState({});
+  const [msg, setMsg] = useState([]);
+
+  const createMessageObj = (id?: string, msg?: string) => {
+    const time = new Date().toString().slice(16, -21).slice(0, 5);
+    setMessageObj({ usr: id, time, msg });
+  };
+
+  const handleInput = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setInput(e.target.value);
+  };
+  const onSubmitForm = (e: React.SyntheticEvent): void => {
+    e.preventDefault();
+    if (!input || !input.trim()) {
```

Comment:
!input 없어도 똑같이 작동합니당
```suggestion
    if (!input.trim()) {
```

Reply:
아하 👍